starter-code/apple-pie-starter-code/index.html

```diff
@@ -1,35 +1,69 @@
 <!DOCTYPE html>
 <html lang="en">
+
 <head>
   <title>Apple Pie Recipe</title>
+  <link rel="stylesheet" type="text/css" href="css/main.css">
+  <link href="https://fonts.googleapis.com/css?family=Lato:300,400,700" rel="stylesheet">
 </head>
-<body>
-Apple Pies
-This was my grandmother's apple pie recipe. I have never seen another one
-quite like it. It will always be my favorite and has won me several first place
-prizes in local competitions. I hope it becomes one of your favorites as well!
-
-Ingredients
-1 recipe pastry for a 9 inch double crust pie
-1/2 cup unsalted butter
-3 tablespoons all-purpose flour
-1/4 cup water
-3 tablespoons all-purpose flour
-1/4 cup water
-1/2 cup white sugar
-1/2 cup packed brown sugar
-8 Granny Smith apples - peeled, cored and sliced
 
-Directions
-Preheat oven to 425 degrees F (220 degrees C). Melt the butter in a saucepan. Stir in
-flour to form a paste. Add water, white sugar and brown sugar, and bring to a boil. Reduce
-temperature and let simmer.
-
-Place the bottom crust in your pan. Fill with apples, mounded slightly. Cover with a
-lattice work crust. Gently pour the sugar and butter liquid over the crust. Pour slowly so
-that it does not run off.
+<body>
+  <header>
+    <h1>Apple Pies</h1>
+  </header>
+  <div class="container">
+    <section class="introduction">
+      <p>This was my grandmother's apple pie recipe. I have never seen another one
+          quite like it. It will always be my favorite and has won me several first place
+          prizes in local competitions. I hope it becomes one of your favorites as well!</p>
+    </section>
+    <hr class="divider">
+    <section class="ingredients">
+      <h2>Ingredients</h2>
+      <ul class="list">
+        <li>- 1 recipe pastry for a 9 inch double crust pie</li>
+        <li>- 1/2 cup unsalted butter</li>
+        <li>- 3 tablespoons all-purpose flour</li>
+        <li>- 1/4 cup water</li>
+        <li>- 1/2 cup white sugar</li>
+        <li>- 1/2 cup packed brown sugar</li>
+        <li>- 8 Granny Smith apples - peeled, cored and sliced</li>
+      </ul>
+      <div class="instructions-container">
+          <ul class="instructions-list">
+            <li>
+              <p>1 h 30m</p>
+              <img src="/images/time.png" alt="time">
```

Comment:
to get these images to work, you need to remove the "/" 

`            <img src="images/time.png" alt="time">
`